### PR TITLE
README now actually contains information

### PR DESCRIPTION
**Pull request by titaniumbones**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,30 @@
...
+
+To add a new tool to this repository, clone or branch the repo and create a new directory for the tool. This directory should include:
+- a `UNIX man`-style README with usage instructions and also a succinct list of any dependencies (shell environment, ruby/python packages, etc.)
+- a main executable file
+- any other files required in order to run the tool
+
+### Tool Specifications
+
+Each tool should:
+* set relevant global variables `path` and `???` in an immediately obvious way at the top of the script, or should provide an obvious way to set those variable through CLI arguments.
```
Comment (dcwalk) [inline README.md:57]: Maybe we add examples for this?

**Review by b5 — APPROVED** [review]
Lovely! Ideally we could have a preamble about any language being fine to use, and maybe a preamble that classifies the different types of harvesting challenges? We could then start to lift the documentation about different types of harvesting challenges into a formal classification. Orrrrrrr we could do that work somewhere else :)

**Comment by titaniumbones** [comment]
Excellent, let's do it here, I can get to it tmrw am i think or late tonight if I'm sleepless. 

On January 31, 2017 7:25:08 PM EST, b5 <notifications@github.com> wrote:
>b5 approved this pull request.
>
>Lovely! Ideally we could have a preamble about any language being fine
>to use, and maybe a preamble that classifies the different types of
>harvesting challenges? We could then start to lift the documentation
>about different types of harvesting challenges into a formal
>classification. Orrrrrrr we could do that work somewhere else :)
>
>
>
>-- 
>You are receiving this because you authored the thread.
>Reply to this email directly or view it on GitHub:
>https://github.com/edgi-govdata-archiving/harvesting-tools/pull/1#pullrequestreview-19459172

-- 
Sent from my Android device with K-9 Mail. Please excuse my brevity.

File: README.md [hunk 2]
```diff
@@ -1,2 +1,30 @@
...
+* write all harvested data to a `../data` directory
+
+I guess it would be cool if tools ALL read the relevant variables from a `paths.txt` or `url.txt` file. Then potentially users wouldn't have to modify the tools at all -- the `zip starter` could write the URL to the appropriate location, and the user could immediately run the script.
```
Comment (dcwalk) [inline README.md]: I think this would be cool, but I think we need to leave room for weird stuff

**Comment by dcwalk** [comment]
@b5 RE: classification, is that the ["Acquire data" section](https://github.com/edgi-govdata-archiving/workflow/tree/master/harvesting-toolkit#4-acquire-the-data) from the workflow? Or something else?

**Comment by b5** [comment]
I think that's heading in the right direction, but in more concrete terms with a higher granularity. This might be overkill, but if we start with stuff like "paginated json api", "xml api", "flash-based content", or "map-driven interface", I'm hoping over time we could use that system to start to understand what tools would be more worth our time. We'll always have an answer of "it's custom", but I think writing down the types of challenges we keep encountering in a systematic manner would be helpful. 

Bu two your point, I think the "acquire data" section is exactly where to start with this. The best place to start would be straight copy-pasting the "Acquire data" section, come up with a naming convention that outlines the exact surface area they cover, and then list all the things we've encountered that these tools _don't_ cover. This'll make for a nice list of things we could ask people to pitch in on ;)

**Comment by titaniumbones** [comment]
@dcwalk @b5 sorry for delay, I made some changes that start to address your comments. Is this good enough to rebase/squash/pull & merge?

**Comment by titaniumbones** [comment]
objections to merging, or should I do it?

**Comment by dcwalk** [comment]
I think it looks great for now!

**Pull request merged**